test/Directory.Build.props

```diff
@@ -10,9 +10,9 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <!-- Using xunit.core and .assert instead of the main package because compilation fails due to warnings triggered by xunit.analyzers. -->
     <!-- <PackageReference Include="xunit" Version="$(XunitPackageVersion)" /> -->
-    <PackageReference Include="xunit.core" Version="2.4.1-pre.build.4059" />
-    <PackageReference Include="xunit.assert" Version="2.4.1-pre.build.4059" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1-pre.build.4059" />
+    <PackageReference Include="xunit.core" Version="2.4.1" />
+    <PackageReference Include="xunit.assert" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
```

Comment:
Does the main `xunit` package still not work? Also `Microsoft.NET.Test.Sdk` looks out of date.

Reply:
For xunit.core and xunit.assert I'm just following the upstream practice. Using the main package generates various build errors because of the analyzers - this can (and probably should) be fixed but we shouldn't block on it...

I've updated Microsoft.NET.Test.Sdk to the latest though, thanks.